core/templates/dev/head/App.ts

```diff
@@ -157,6 +157,7 @@ angular.module('oppia').config([
                 rejection.config.url) : '';
               var additionalLoggingInfo = warningMessage +
                 '\n URL: ' + rejectionUrl +
+                '\n stacktrace: ' + rejection.data.stacktrace +
```

Comment:
Hm, will you be able to see this in Travis? I guess yes because we can see the console logs after every test, right?

(Unfortunately, they seem to be passing... :( )

Reply:
Yep it should show up. I checked locally.

haha yes, I think we will have to do that. I'll push empty commits to restart the tests.